app/models/user.rb

```diff
@@ -23,17 +23,41 @@ def affiliations
     @affiliations ||= ENV['suAffiliation']
   end
 
+  def person_affiliations
+    @person_affiliations ||= ENV['unscoped-affiliation']
+  end
+
+  # If a user has proper Stanford affiliation to allow access to resources now depends
+  # on two kinds of affiliation information: the 'su_affiliated' method which returns true
```

Comment:
Do we need to check both places (e.g. are there people with an `su_affiliation` group that grants access that `unscoped-affiliation` member doesn't), or is this a temporary thing for migration?

Reply:
This is a good question. I'll follow up with Sarah and Irina.  I know the reverse is true (i.e. unscoped-affiliation grants access su_affiliation doesn't) but not sure about this question.

Reply:
I also didn't want to mess with that part of the logic because the su_affiliation allowed list is quite long.